salt/master.py

```diff
@@ -226,7 +226,9 @@ def run(self):
 
         # Make Start Times
         last = int(time.time())
+        last_git_pillar = last
 
+        git_pillar_interval = self.opts.get('git_pillar_interval', 0)
```

Comment:
Is this intended to be a user-configurable option? If so, it needs documentation. :)